### PR TITLE
Add slides for Zarr talk and tools plenary sessions

### PR DESCRIPTION
**Pull request by sanketverma1704**
Hi! 👋🏻 

This PR adds the slides for the talk, "Zarr: Community specification of large, cloud-optimised, N-dimensional, typed array storage". 

Link: https://cfp.scipy.org/2023/talk/T3NSL8/

CC: @joshmoore @jakirkham

### CODE REVIEW

**Comment by jakirkham** [comment]
@joshmoore should we also add the slides from the tools plenary?

File: presentations/slides/zarr/info.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            "orcid": "0000-0003-4028-811X"
+        },
+        {
+            "name": "John Kirkham",
+            "affiliation": "NVIDIA",
+            "orcid": "0009-0007-6130-9705"
+        }
+
+    ],
+    "description": "A key feature of the Python data ecosystem is the reliance on simple but efficient primitives that follow well-defined interfaces to make tools work seamlessly together (Cf. http://data-apis.org/). NumPy provides an in-memory representation for tensors. Dask provides parallelisation of tensor access. Xarray provides metadata linking tensor dimensions. Zarr provides a missing feature, namely the scalable, persistent storage for annotated hierarchies of tensors. Defined through a community process, the Zarr specification enables the storage of large out-of-memory datasets locally and in the cloud. Implementations exist in C++, C, Java, Javascript, Julia, and Python, enabling."
```
Comment (jakirkham) [inline presentations/slides/zarr/info.json]:
```suggestion
    "description": "A key feature of the Python data ecosystem is the reliance on simple but efficient primitives that follow well-defined interfaces to make tools work seamlessly together ( Cf. https://data-apis.org/ ). NumPy provides an in-memory representation for tensors. Dask provides parallelisation of tensor access. Xarray provides metadata linking tensor dimensions. Zarr provides a missing feature, namely the scalable, persistent storage for annotated hierarchies of tensors. Defined through a community process, the Zarr specification enables the storage of large out-of-memory datasets locally and in the cloud. Implementations exist in C++, C, Java, Javascript, Julia, and Python, enabling."
```

**Comment by joshmoore** [comment]
Here are the update slides (I won't be able to update this branch)

[2023-Moore-Zarr-SciPy-TechUpdate.pdf](https://github.com/scipy-conference/scipy_proceedings/files/12167059/2023-Moore-Zarr-SciPy-TechUpdate.pdf)

**Comment by sanketverma1704** [comment]
Thanks for the nudge, @jakirkham and for sharing the slides, @joshmoore. I've updated the branch.

**Comment by jakirkham** [comment]
Thanks Sanket! 🙏

Not sure if the presentations need to be in different directories. Maybe a reviewer can let us know

**Review by deniederhut — APPROVED** [review]
giphy-567286036

**Comment by jakirkham** [comment]
Thanks Dillon! 🙏

**Comment by deniederhut** [comment]
🫡

**Pull request merged**